### PR TITLE
Task/703 capability display with decoded json

### PR DESCRIPTION
**Pull request by ywkim312**
## Description
There are some duplicate codes in contributions and catalog since the packaging the code was not completed. This should be refactored and cleaned up when the actual packing gets implement. 

Please add these variables in .env of the catalog. These are duplicated to catalog but they are needed to run the application.
ADMIN_USERS = sandeep-ps,ywkim312

**Fixes #(add issue number here and remove parentheses)**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [x] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [x] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [x] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: contributions/catalog/controllers/config.py [hunk 1]
```diff
@@ -24,3 +38,4 @@ class Config(object):
     AUTHORIZATION_BASE_URL = os.getenv("AUTHORIZATION_BASE_URL", 'https://github.com/login/oauth/authorize')
     TOKEN_URL = os.getenv("TOKEN_URL", 'https://github.com/login/oauth/access_token')
     CORS_ENABLED = bool(os.getenv('CORS_ENABLED', 'False') == 'True')
+    ADMIN_USERS = os.getenv('ADMIN_USERS', [])
```
Comment (sandeep-ps) [inline contributions/catalog/controllers/config.py:41]:
```suggestion
    ADMIN_USERS = os.getenv('ADMIN_USERS', '')
```
I think the default value should be the empty string as this config expects a string literal.

Comment (sandeep-ps) [reply contributions/catalog/controllers/config.py:41]: I have the feedback for Contributions configs.py.

**Comment by sandeep-ps** [comment]
The rest looks good to me. Thanks.

File: contributions/catalog/controllers/config.py (hunk 1 above)
Comment (ywkim312) [reply contributions/catalog/controllers/config.py]: you're right. must be confused with contributionAdmins. Thanks for catching them.

**Review by sandeep-ps — APPROVED** [review]

**Pull request merged**